app/controllers/video_controller.rb

```diff
@@ -0,0 +1,15 @@
+class VideoController < ApplicationController
+  def index
+    @filterrific = initialize_filterrific(
```

Comment:
Does this variable ever get referenced by views? If not, no need to set it as instance variable.